tests/test_aaa_authorization_service.rb

```diff
@@ -22,54 +22,72 @@ class TestAaaAuthorizationService < CiscoTestCase
   def setup
     super
     feature_tacacs
+    preconfig_tacacs_server_access(tacacs_groups[0])
+    config_tacacs_servers(tacacs_groups[1..3])
   end
 
   def teardown
+    cleanup_aaa
     feature_tacacs(false)
```

Comment:
I think I'll remove the feature disablement instead since I don't think it's helpful in cleanup.

Removing it decreases test time from 228.7 seconds to 92.8 seconds.